ddtrace/_trace/tracer.py

```diff
@@ -283,9 +282,7 @@ def __init__(
             register_on_exit_signal(self._atexit)
 
         self._hooks = _hooks.Hooks()
-        atexit.register(self._atexit)
```

Comment:
We should continue to register these hooks here. We can make the start/restart/stop/at_exit methods in ddtrace/_trace/tracer.py no-ops for now. We can migrate these hooks in v4.0